CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - On Windows, query DirectWrite for recommended anti-aliasing settings
+- GPU memory usage has been decreased by disabling allocation of depth and stencil buffers
```

Comment:
I think I'd move this to the `fixed` section, usually `changed` is something that we intentionally did one way and now do another way. If we would have done it differently from the beginning if we would have known, it's a fix.